spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -232,6 +233,16 @@ def main(argv: Sequence[str]):
                 if contrast != "stir":
                     parser.error("Task 'seg_sc_ms_lesion_stir_psir' requires the flag `-c` to be either psir or stir.")
 
+        if 'seg_sc_epi' in arguments.task[0]:
+            for image in arguments.i:
+                image_load = nb.load(image)
+                image_shape = image_load.shape
+                if len(image_shape) == 4:
+                    parser.error("Only 3D vloumes supported for this task. You can either provide a mean volume or a single time point")
+                else:
+                    input_filenames = arguments.i.copy()
```

Comment:
Is there a reason for duplicating this line, since `input_filenames` is already created earlier? I'm wondering, if we exclude it, will the processing be basically the same?

Reply:
Let me check on this

Reply:
Again, that sounds perfect. Removed this!

Reply:
Ah, oops! It looks like the wrong one might have been removed, as we have a new error now: 

```
UnboundLocalError: local variable 'input_filenames' referenced before assignment
```

Reply:
Hmm, it works on my end. Do you still get the error?

Reply:
The error occurs in the test suite for non-EPI models, hehe: https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/8822601995/job/24221178754?pr=4454#step:5:728

Reply:
Oh yeah, found it!